api/app/controllers/spree/api/base_controller.rb

```diff
@@ -100,10 +100,27 @@ def invalid_resource!(resource)
       end
 
       def api_key
-        request.headers["X-Spree-Token"] || params[:token]
+        bearer_token || spree_token || params[:token]
       end
       helper_method :api_key
 
+      def bearer_token
+        pattern = /^Bearer /
+        header  = request.headers["Authorization"]
```

Comment:
I'd remove one space after `header`, it's aligned to `pattern` on the line above but if that line changes in the future we'll be forced to change this one as well and this is not ideal for git history. What do you think?